.github/workflows/docker-and-dependencies.yml

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: "Static and dependency tests"
+on: [pull_request, push]
+
+jobs:
+  create-production-docker-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+    steps:
+      - uses: actions/checkout@v2
+        name: checkout
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: install airflow
+        run: pip install -e .[devel]
+      - name: build docker image
+        run: ./breeze build-image --production-image
+        env:
+         PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
+  generate-requirements:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6,3.7]
+    steps:
+      - uses: actions/checkout@v2
+        name: checkout
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: install airflow
+        run: pip install -e .[devel]
+      - name: generate requirements
+        run: ./scripts/ci/ci_generate_requirements.sh
+        env:
+          PYTHON_MAJOR_MINOR_VERSION: ${{ matrix.python-version }}
+          SHOW_GENERATE_REQUIREMENTS_INSTRUCTIONS: true
+  prepare-and-test-backport-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: checkout
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: install airflow
+        run: pip install -e .[devel]
+      - name: Prepare & test backport packages 1.10.9
+        run: ./scripts/ci/ci_prepare_and_test_backport_packages.sh
```

Comment:
- name: "Before install"
        run: pip install bowler
      - name: "Prepare backport packages"
        run: ./scripts/ci/ci_prepare_backport_packages.sh

Reply:
👍 

@turbaszek I'm gonna try to get JUST static checks + dependency + docker and merge that to at minimum reduce pressure on travis.

Reply:
Feel free to take some from my PR. Static checks, docs and test (without kube) should work

Reply:
I had to run `pre-commit run update-setup-cfg-file --all-files` to make static checks work